addons/api/mirage/scenarios/default.js

```diff
@@ -13,6 +13,8 @@ export default function(server) {
     scope: { id: globalScope.id, type: globalScope.type }
   }, 'withChildren')[0];
 
+  server.create('user', { id: 'authenticateduser' });
```

Comment:
Is this necessary?

Reply:
When responding to sessions query, I'm updating userId of alternate session records to `authenticateduser`. Due to this association, the actual user object needs to exist otherwise mirage throws a ` Mirage: Couldn't find user with id = authenticateduser Error: Mirage: Couldn't find user with id = authenticateduser`